internal/server/models/identity.go

```diff
@@ -43,3 +47,11 @@ func (i *Identity) ToAPI() *api.User {
 func (i *Identity) PolyID() uid.PolymorphicID {
 	return uid.NewIdentityPolymorphicID(i.ID)
 }
+
+func (i *Identity) BeforeSave(_ *gorm.DB) error {
+	if len(i.VerificationToken) == 0 {
+		i.VerificationToken = generate.MathRandom(10, generate.CharsetAlphaNumeric)
```

Comment:
Are there two ways to generate a token? The default value for the database field, and this here?

Do we need both?  Doing it in the application (this code here), seems a little easier to read than the default on the database field.